eZ/Publish/Core/Repository/ContentService.php

```diff
@@ -1742,6 +1742,21 @@ protected function copyTranslationsFromPublishedVersion(APIVersionInfo $versionI
         $this->internalUpdateContent($versionInfo, $updateStruct);
     }
 
+    protected function isHashEqual($fieldType, $newValue, $fieldValue): bool
```

Comment:
Missing typehints.

Reply:
is not possible type hints, it depends on fieldType

Reply:
Well, it could be typehinted to
`\eZ\Publish\Core\FieldType\FieldType` and `eZ\Publish\SPI\FieldType\Value`